packages/web-components/src/keplr-connection/followLatest.js

```diff
@@ -0,0 +1,48 @@
+// @ts-check
+import { iterateEach } from '@agoric/casting';
+import { AmountMath } from '@agoric/ertp';
+
+export const followLatest = async ({startingHeight, latestFollower, updatePurses, brandToPurse}) => {
```

Comment:
```suggestion
// TODO: Provide a way for the caller to terminate the follower loop.
export const followLatest = async ({startingHeight, latestFollower, updatePurses, brandToPurse}) => {
```

Reply:
Done. Is there a good way to cancel a follower? Maybe set a flag and break on the next update?